EventsExpress/ClientApp/src/actions/register/register-complete-action.js

```diff
@@ -1,32 +1,47 @@
+import { SubmissionError } from 'redux-form';
+import moment from 'moment';
+import jwt from 'jsonwebtoken';
 import { AuthenticationService } from '../../services';
-import { createBrowserHistory } from 'history';
-import { setSuccessAllert } from '../alert-action';
+import { setSuccessAllert, setErrorAllertFromResponse } from '../alert-action';
 import { buildValidationState } from '../../components/helpers/action-helpers';
-import { SubmissionError } from 'redux-form';
 import { jwtStorageKey } from '../../constants/constants';
-import jwt from 'jsonwebtoken';
+import { getUserInfo } from '../login/login-action';
 
 const api_serv = new AuthenticationService();
-const history = createBrowserHistory({ forceRefresh: true });
 
-export default function registerComplete(data) {
+export default function registerComplete(data, { shouldSaveMoreInfo }) {
```

Comment:
Are there cases when we are not saving additional info?

Reply:
We are only saving additional info when user clicks "Confirm" on the final step, otherwise we save only profile data